README.md

```diff
@@ -229,17 +229,11 @@ Then you want to log into the Spot CAM over the browser. In your browser, type i
 
 The default port for SDP is 31102 for the Spot CAM. Once inside, you will be prompted to log in using your username and password. Do so and the WebRTC frames should begin to properly stream.
 
+# Spot ROS 2 Control
+This repository also provides a ROS 2 control package designed to interface with Spot's joint control API. Further documentation and examples can be found in [`spot_ros2_control`](./spot_ros2_control/).
 
 # Advanced Install
 
-## Install spot_msgs as a deb package
-`spot_msgs` are normally compiled as part of this repository.  If you would prefer to install them as a debian package, follow the steps below:
-```bash
-wget -q -O /tmp/ros-humble-spot-msgs_0.0.0-0jammy_amd64.deb https://github.com/bdaiinstitute/spot_ros2/releases/download/spot_msgs-v0.0-0/ros-humble-spot-msgs_0.0.0-0jammy_amd64.deb
-sudo dpkg -i /tmp/ros-humble-spot-msgs_0.0.0-0jammy_amd64.deb
-rm /tmp/ros-humble-spot-msgs_0.0.0-0jammy_amd64.deb
-```
-
```

Comment:
Deleted this section as this release is super out of date and we shouldn't be pointing people to it